zipline/assets/asset_db_schema.py

```diff
@@ -84,6 +85,29 @@
     ),
 )
 
+supplementary_mappings = sa.Table(
+    'supplementary_mappings',
+    metadata,
+    sa.Column(
+        'id',
+        sa.Integer,
+        unique=True,
+        nullable=False,
+        primary_key=True,
+    ),
+    sa.Column(
+        'sid',
+        sa.Integer,
+        sa.ForeignKey(equities.c.sid),
+        nullable=False,
+        index=True,
```

Comment:
Probably not, maybe we should change this on the `equity_symbol_mappings` table as well? https://github.com/quantopian/zipline/pull/1612/files#diff-f5e9723ed70ad2f79bc4adb3ba2d15d8R60